sonar-project.properties

```diff
@@ -3,7 +3,7 @@ sonar.projectKey=SAP_open-ux-tools
 sonar.sources=.
 sonar.inclusions=**/*.ts, **/*.tsx
 sonar.exclusions=**/*.test.ts, **/*.test.tsx, **/*.story.tsx, **/examples/**/*, **/templates/**/*, **/test/**/*, **/jest-file-matchers/src/matchers/types.ts, **/jest-runner-puppeteer/**/*
-sonar.cpd.exclusions=**/i18n.ts, **/fiori-elements-writer/src/packageConfig.ts, **/fiori-freestyle-writer/src/packageConfig.ts, **/ui5-info/src/ui5-version-fallback.ts, **/odata-vocabularies/src/resources/*
+sonar.cpd.exclusions=**/i18n.ts, **/fiori-elements-writer/src/packageConfig.ts, **/fiori-freestyle-writer/src/packageConfig.ts, **/ui5-info/src/ui5-version-fallback.ts, **/odata-vocabularies/src/resources/*, **/ui5-library-reference-inquirer/src/prompts/helpers.ts
```

Comment:
This can be removed when the TBI https://github.com/SAP/open-ux-tools/issues/1819 is addressed